regression-test/suites/inverted_index_p0/test_index_approx_top_k.groovy

```diff
@@ -88,15 +88,11 @@ suite("test_index_approx_top_k", "p0"){
 
         sql """ set debug_skip_fold_constant = true; """
         qt_sql """ select clientip, count(*) as count from ${tableName} group by clientip order by count desc, clientip asc limit 10; """
-        qt_sql """ select approx_top_k(clientip) from ${tableName}; """
-        qt_sql """ select approx_top_k(clientip, 10) from ${tableName}; """
```

Comment:
May I ask why deleting these tests?

Reply:
Currently, it is difficult to implement default values for the variable arguments in the function framework.  It is hoped that users will directly specify the last two parameters.